### PR TITLE
mute terminal hint color

### LINKED ISSUE #213647: Terminal: Inline chat recommendation is too distracting
Version: 1.90.0-insider
Commit: 50f2b2eca8d5c6e19ec95f9d46a59f623e60cac3
Date: 2024-05-28T05:49:03.008Z
Electron: 29.4.0
ElectronBuildId: 9593362
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Linux x64 5.14.0-1048-oem

- Have copilot installed and open the terminal

image

The inline chat recommendation uses the same colors as the terminal prompt and I find it too distracting.
Would be better if it uses the color that ghost text in the editor uses.

### PR DESCRIPTION
**Pull request by meganrogge**
fix #213647

### CODE REVIEW

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts [hunk 1]
```diff
@@ -120,6 +121,12 @@ export const TERMINAL_TAB_ACTIVE_BORDER = registerColor('terminal.tab.activeBord
 	hcDark: TAB_ACTIVE_BORDER,
 	hcLight: TAB_ACTIVE_BORDER
 }, nls.localize('terminal.tab.activeBorder', 'Border on the side of the terminal tab in the panel. This defaults to tab.activeBorder.'));
+export const TERMINAL_INITIAL_HINT_FOREGROUND = registerColor('terminal.initialHint.foreground', {
+	dark: Color.fromHex('#ffffff56'),
+	light: Color.fromHex('#0007'),
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts:126]: I see we don't use this `Color.fromHex` for other colors 🤔 . I'm wondering why 👀

Comment (meganrogge) [reply src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts:126]: looks like that's not necessary, removing

File: src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts [hunk 2]
```diff
@@ -120,6 +120,12 @@ export const TERMINAL_TAB_ACTIVE_BORDER = registerColor('terminal.tab.activeBord
 	hcDark: TAB_ACTIVE_BORDER,
 	hcLight: TAB_ACTIVE_BORDER
 }, nls.localize('terminal.tab.activeBorder', 'Border on the side of the terminal tab in the panel. This defaults to tab.activeBorder.'));
+export const TERMINAL_INITIAL_HINT_FOREGROUND = registerColor('terminal.initialHint.foreground', {
+	dark: '#ffffff56',
+	light: '#0007',
+	hcDark: '#ffffff56',
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts:126]: seeing what we should do for HC themes atm

File: src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts [hunk 3]
```diff
@@ -120,6 +120,12 @@ export const TERMINAL_TAB_ACTIVE_BORDER = registerColor('terminal.tab.activeBord
 	hcDark: TAB_ACTIVE_BORDER,
 	hcLight: TAB_ACTIVE_BORDER
 }, nls.localize('terminal.tab.activeBorder', 'Border on the side of the terminal tab in the panel. This defaults to tab.activeBorder.'));
+export const TERMINAL_INITIAL_HINT_FOREGROUND = registerColor('terminal.initialHint.foreground', {
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts:123]: `terminal.initialHintForeground` would be more consistent with others (I know it's messy across the product but we want it to be as consistent as possible within the component.

**Review by Tyriar — DISMISSED** [review]

**Review by Tyriar — APPROVED** [review]

**Pull request merged**